README.md

```diff
@@ -165,11 +175,12 @@ var rnabtest = React.createClass({
       100,
       experiment
     );
+
     ga.send(gaEvent);
   }
 
-  _addImpression() {
-    var gaImpression = new GAHits.Impression(
+  _productImpression() {
+    var gaProductImpression = new GAHits.Impression(
```

Comment:
I am wandering what's the difference between `GAHits.Impression` and `GAHits.Product`. Then seems to be all for products

Reply:
Good question the main difference is that they differ on the information they collect regarding a product. Also, the action they represent are different. In the links bellow you will be able to see the differences.

https://developers.google.com/analytics/devguides/collection/analyticsjs/enhanced-ecommerce#impression-data

https://developers.google.com/analytics/devguides/collection/analyticsjs/enhanced-ecommerce#product-data

Reply:
👍